packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_loan_withdraw_from_vault.test.ts

```diff
@@ -693,6 +693,7 @@ describe('withdrawFromVault with 50% DUSD or DFI collaterals', () => {
     expect(outs[1].value).toBeGreaterThan(9.999)
     expect(outs[1].scriptPubKey.addresses[0]).toStrictEqual(bobVaultAddr)
 
+    await alice.generate(1)
```

Comment:
Is this necessary? Bob is doing the withdrawal and the 1 block is minted and `waitForSync` should suffice.

Reply:
From my testing, the tests behave more consistently when I added this line. Can't deny that it might be coincidence or other factor, but it somehow worked.

Reply:
I'll revert this, and see what happens. If it becomes inconsistent again, i'll look into the reason.